avocado/core/loader.py

```diff
@@ -34,6 +34,7 @@
 from ..utils import stacktrace
 from .settings import settings
 from .output import LOG_UI
+from .test import DryRunTest
```

Comment:
There's already an import of the module `test` on line 31, so you can just reuse it.

Reply:
Yes, of course